build/net/src/lib.rs

```diff
@@ -27,7 +27,8 @@ pub struct NetConfig {
     /// VLAN configuration, or None. This is checked against enabled features
     /// during the `net` build, so it must be present iff the `vlan` feature
     /// is turned on.
```

Comment:
nit - the doc comment still refers to this as optional

Reply:
Updated to "must be non-empty" in 23b0a914ef2a